File: a_plus_b.py
'''
# Sample code to perform I/O:

name = input()                  # Reading input from STDIN
print('Hi, %s.' % name)         # Writing output to STDOUT

# Warning: Printing unwanted or ill-formatted data to output will cause the test cases to fail
'''

# Write your code here
while(True):
    try:
        ln=input()
        lst=ln.split(" ")
        res=int(lst[0]) + int(lst[1])
        print(res)
    except EOFError:
        break

File: APlusB.java
/* IMPORTANT: Multiple classes and nested static classes are supported */

/*
 * uncomment this if you want to read input.
//imports for BufferedReader
import java.io.BufferedReader;
import java.io.InputStreamReader;

//import for Scanner and other utility classes

*/

// Warning: Printing unwanted or ill-formatted data to output will cause the test cases to fail
import java.util.*;
class APlusB {
    public static void main(String args[] ) throws Exception {
        /* Sample code to perform I/O:
         * Use either of these methods for input

        //BufferedReader
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String name = br.readLine();                // Reading input from STDIN
        System.out.println("Hi, " + name + ".");    // Writing output to STDOUT

        //Scanner
                         // Reading input from STDIN
        System.out.println("Hi, " + name + ".");    // Writing output to STDOUT

        */

        // Write your code here

        Scanner s = new Scanner(System.in);
        int a,b,i=1;
        while(i<=3){
            //s.nextLine();
            a = s.nextInt();
            
            b = s.nextInt();
            //String line=s.nextLine();
            System.out.println(a+b);
            i++;
            /*if(line.equals("")){
                System.out.println("HELLO");
                break;
            }*/
        }

    }
}
